### PR TITLE
Feature/assign task to member

### PR DESCRIPTION
**Pull request by PaulJohannFlade**

### CODE REVIEW

**Review by NikRoe — CHANGES_REQUESTED** [review]
Hi there,

nice start!

I left a comment regarding the `assignedTo` key in your objects. In the current state of the application is does not make sense for this to be an array, but it makes more sense to be a string.

The code you write for a feature to implement should only cover what this feature currently needs, not paving the way for features that might come next.
Once you can select multiple persons to assign a task to, it makes sense to change the data type to be an array. But not before that.

Keep it up 🙌

File: components/TaskDetails.js [hunk 1]
```diff
@@ -121,6 +122,14 @@ export default function TaskDetails({
             onChange={() => onCheckboxChange(id)}
           />
         </label>
+        <p>Member:</p>
+        <h2>
+          {assignedTo.map((id) => (
```
[the remark this answers is not in the document]
Comment (PaulJohannFlade) [reply components/TaskDetails.js]: Thanks for the review! The changes are done. :-)

File: components/Form.js [hunk 2]
```diff
@@ -61,9 +66,9 @@ export default function Form({ onTaskSubmit, title, value, isEdit }) {
     }
 
     if (isEdit) {
-      onTaskSubmit({ ...data, id: value.id });
+      onTaskSubmit({ ...data, id: value.id, isDone: value.isDone });
     } else {
-      onTaskSubmit(data);
+      onTaskSubmit({ ...data });
```
Comment (NikRoe) [inline components/Form.js:71]: This change should not be needed. You are currently passing a new object as the argument and spread `data` to that new object.
In the end, whether the argument is `(data)` or `({ ...data })` it makes no difference in the data type. Both are objects, but going with `({ ...data })` makes the code more complicated than it needs to be.

**Review by NikRoe — APPROVED** [review]
Hi there,

nicely done 🙌 

I left one comment regarding the else-branch in your form's `handleSubmit`. 

Keep it up 👍

**Pull request merged**